### ISSUE TITLE
Visualization of sample model not working

### ISSUE THREAD

**Issue by gabrielebndn** [stack-of-tasks/pinocchio#726]
The following minimal example
```
import pinocchio as pin

model = pin.buildSampleModelHumanoid()
geom_model = pin.buildSampleGeometryModelHumanoid(model)
q = pin.neutral(model)

robot = pin.RobotWrapper(model,geom_model,geom_model)

raw_input("initDisplay")
robot.initDisplay()

raw_input("loadDisplayModel")
robot.loadDisplayModel()

raw_input("display")
robot.display(q)

print("done")
```
is not working. The model is not loaded inside gepetto-gui and it does not show up.
However, gepetto-gui itself seems to be fine (`examples/python/gepetto-viewer.py`, which loads DAE files for romeo, is working for me).

On `loadDisplayModel()`, I get a bunch of errors like this
```
 does not exist.: iostream error
```
followed by errors that look like
```
Node "world/pinocchio/collisions/rlegshoulder_object" does not exist.
```

I don't know where this problem might come from. Am I doing this right? Is there something wrong in `buildSampleGeometryModelHumanoid`? Can this be a probem of hpp-fcl instead of Pinocchio?

**Comment by jcarpent** [comment]
I don't think that hpp-fcl basic geometries are parsed by the viewer.

**Comment by gabrielebndn** [comment]
I don't think there is any parsing involved. Plus, I remember @nmansard made the `buildSampleGeometryModel` methods, and they were working on gepetto-viewer. If it does not, I think it should. @nmansard, may you verify this?

**Comment by nmansard** [comment]
robot wrapper does not create the simple geometry. And it cannot do it because we don't have the bindings for the simple geometries.

I started to map the simple geometries to the GUI, but manually. And I stopped in the middle.

@jcarpent I think you told me that you had some piece of code to expose hpp-fcl simple geometries. I believe we even have an issue for that open since a long time.

**Comment by gabrielebndn** [comment]
The issue was #548, but it was closed, I don't know why.

**Comment by jcarpent** [comment]
I'm not able to find them. They seem to be on my previous laptop, which has been reformated since then :(

**Comment by gabrielebndn** [comment]
Solved by #1004
